single_stage_detector/README.md

```diff
@@ -30,9 +30,35 @@ cd reference/single_stage_detector/
 source download_dataset.sh
 ```
 
-### Steps to run benchmark.
+## Running with Popper
 
-## Steps to launch training
+[Popper](https://github.com/systemslab/popper) is a tool for defining and executing container-native workflows either locally or on CI services. We use Popper to convert the benchmarks into reproducible workflows. More details about Popper can be found [here](https://popper.readthedocs.io/).
+
+### Instructions:
+
+1. Clone the repository.
+```
+git clone https://github.com/mlperf/training
+```
+
+2. Install docker, cuda-runtime and nvidia-docker on the machine.
+```
+./training/install_cuda_docker.sh
+```
+
+3. Install the `popper` tool.
+```
+pip install popper
+```
+We recommend to use a [virtualenv](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment) for installing Popper.
+
+4. Run the workflow.
+```
+cd single_stage_detector/
+popper run -f main.yml -c settings.py
+```
+Here, the `settings.py` file contains necessary configuration that needs to be passed to the container engine
```

Comment:
"to the container engine in order to use the nvidia drivers. For more information about customizing container engine parameters, see [here](https://popper.readthedocs.io/en/latest/sections/cli_features.html#customizing-container-engine-behavior)."